lib/board.rb

```diff
@@ -0,0 +1,56 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
```

Comment:
Why not just use the `#reset!` method you built in `#initialize` ?  Using methods like that rather than repeating code helps keep our code DRY